workshops/real_numbers_and_strings.py

```diff
@@ -1,8 +1,9 @@
 def interest_rate(interest_rate, rubles, kopecks):
     k = (rubles * 100 + kopecks) * (100 + interest_rate) // 100
-    return k//100, k%100
+    return k//100, k % 100
```

Comment:
I don't understand why % needs spaces and // doesn't need them. Probably they should be the same.

Reply:
According to pep8 % is low priority operation

Reply:
I added the comment because % and // have the same priority in Python. So I assume they should also have the same spacing. pep8 doesn't really mention % as a specific operation, it only says "add spaces around operators with lower priority, and don't add them around operators with higher priority, but use your own judgement for this".